### ISSUE TITLE
SELECT SQL execute failed in standalone mode

### ISSUE THREAD

**Issue by dl239** [4paradigm/OpenMLDB#806]
```
CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double,dropoff_latitude double, store_and_fwd_flag string,trip_duration int, INDEX(key=vendor_id, ts=pickup_datetime), INDEX(key=passenger_count, ts=pickup_datetime));
```

```
SELECT trip_duration, passenger_count,
sum(pickup_latitude) OVER w AS vendor_sum_pl,
max(pickup_latitude) OVER w AS vendor_max_pl,
min(pickup_latitude) OVER w AS vendor_min_pl,
avg(pickup_latitude) OVER w AS vendor_avg_pl,
sum(pickup_latitude) OVER w2 AS pc_sum_pl,
max(pickup_latitude) OVER w2 AS pc_max_pl,
min(pickup_latitude) OVER w2 AS pc_min_pl,
avg(pickup_latitude) OVER w2 AS pc_avg_pl,
count(vendor_id) OVER w2 AS pc_cnt,
count(vendor_id) OVER w AS vendor_cnt
FROM t1
WINDOW w AS (PARTITION BY vendor_id ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW) INTO OUTFILE '/tmp/feature.csv';
```

![wecom-temp-4c406ae2b47017759f063a15f862a5b9](https://user-images.githubusercontent.com/6183996/144176864-ec0b56fd-46f4-43bc-b642-929769aa8cf3.png)

**Comment by dl239** [comment]
![WX20211201-142003](https://user-images.githubusercontent.com/6183996/144182470-61cfa2f1-634b-42c5-8f49-2544339f0138.png)

**Comment by aceforeverd** [comment]
yaml case
```yaml
  - id: 2
    inputs:
      -
        name: t1
        columns:
          - id string
          - vendor_id int
          - pickup_datetime timestamp
          - dropoff_datetime timestamp
          - passenger_count int
          - pickup_longitude int
          - pickup_latitude double
          - dropoff_longitude double
          - dropoff_latitude double
          - store_and_fwd_flag string
          - trip_duration int
        indexs:
          - idx:vendor_id:pickup_datetime
          - idx2:passenger_count:pickup_datetime
    sql: |
      SELECT trip_duration, passenger_count,
      sum(pickup_latitude) OVER w AS vendor_sum_pl,
      max(pickup_latitude) OVER w AS vendor_max_pl,
      min(pickup_latitude) OVER w AS vendor_min_pl,
      avg(pickup_latitude) OVER w AS vendor_avg_pl,
      sum(pickup_latitude) OVER w2 AS pc_sum_pl,
      max(pickup_latitude) OVER w2 AS pc_max_pl,
      min(pickup_latitude) OVER w2 AS pc_min_pl,
      avg(pickup_latitude) OVER w2 AS pc_avg_pl,
      count(vendor_id) OVER w2 AS pc_cnt,
      count(vendor_id) OVER w AS vendor_cnt
      FROM t1
      WINDOW w AS (PARTITION BY vendor_id ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
      w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW);
    expect:
      success: true
```

nested agg window validate fail
```log
PROJECT(type=WindowAggregation)
  +-WINDOW(partition_keys=(vendor_id), orders=(pickup_datetime ASC), range=(pickup_datetime, -86400000, 0))
  PROJECT(type=WindowAggregation, NEED_APPEND_INPUT)
    +-WINDOW(partition_keys=(), orders=(ASC), range=(pickup_datetime, -86400000, 0))
    DATA_PROVIDER(type=Partition, table=t1, index=idx2)
I1201 16:37:02.269774 439047680 engine_test_base.cc:397] SQL Compile take 20.544 milliseconds
I1201 16:37:02.269786 439047680 engine_test_base.cc:400] Isn't partition provider
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/sql_compiler.cc:262)
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/sql_compiler.cc:166)
    (Caused by) Fail to generate physical plan batch mode
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/transform.cc:1727)
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/transform.cc:1505)
    (Caused by) Fail to support physical plan in performance sensitive mode
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/transform.cc:1632)
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/transform.cc:1484)
    (Caused by) Window node hasn't been optimized
    (At /Users/danielace/Git/4paradigm/OpenMLDB/hybridse/src/vm/transform.cc:1451)
    (Caused by) Isn't partition provider
```

**Comment by jingchen2222** [comment]
Multiple windows aggregation is non-support in performance-sensitive mode.
We do have some SQL that can only be executed in online request mode.
Performance-sensitive rules and boundaries are really difficult to understand. I think we should eliminate the concept for users in v0.4.0
https://github.com/4paradigm/OpenMLDB/issues/796

**Comment by jingchen2222** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [bugfix tracking](https://trello.com/c/wEQ9HW0k/25-bugfix-tracking)

**Comment by jingchen2222** [comment]
Resolved by #796 
![image](https://user-images.githubusercontent.com/6291173/148158524-9239fe4a-76d5-41d4-bbf3-a3358f95b7af.png)
